portfolio/src/main/java/com/google/sps/servlets/CheckLoginServlet.java

```diff
@@ -31,26 +31,30 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
           UserService userService = UserServiceFactory.getUserService();
           boolean isloggedin = false;
           String url = "/";
+          String email = "null";
           if(userService.isUserLoggedIn()){
             isloggedin = true;
             String urlToRedirectToAfterUserLogsOut = "/";
             url = userService.createLogoutURL(urlToRedirectToAfterUserLogsOut);
+            email = userService.getCurrentUser().getEmail();
           }
           else{
             String urlToRedirectToAfterUserLogsIn = "/";
             url = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
           }
-          LoginInfo login = new LoginInfo(isloggedin, url);
+          LoginInfo login = new LoginInfo(isloggedin, url, email);
           Gson gson = new Gson();
           response.setContentType("application/json;");
           response.getWriter().println(gson.toJson(login));
   }
   private class LoginInfo{
     boolean loginStatus;
-    String url;
-    LoginInfo(boolean loginStatus, String url){
+      String url;
+      String email;
+    LoginInfo(boolean loginStatus, String url,String email){
```

Comment:
considering loginStatus is a boolean, we can make this isLoggedIn. 
https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig

If you decide to change this, make sure to change it here and everywhere it's used in this PR.